src/components/deck/default-deck.js

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import Deck from './deck';
+import useBroadcastChannel from '../../hooks/use-broadcast-channel';
+
+export default function DefaultDeck(props) {
+  const deck = React.useRef();
+
```

Comment:
Default Deck is just a wrapper that adds the broadcast channel logic to the deck's navigation. We don't want it as part of the internal deck component since those are used on overview and presenter modes and do not need this logic.